[PowerToys] Run page: Additional updates and improvements (#3768)
MicrosoftDocs/windows-uwp: hub/powertoys/run.md

| Ignore shortcuts in Fullscreen mode | When in full-screen (F11), PowerToys Run won't be engaged with the shortcut |
| Maximum number of results | Maximum number of results shown without scrolling |
| Clear the previous query on launch | When launched, previous searches will not be highlighted |

> | Preferred monitor position | If multiple monitors are in use, PowerToys Run can be launched on the desired monitor:<br />- Primary display<br />- Display with mouse cursor<br />- Display with focused window |

Author: Hopefully this fixes translation

Reviewer: If translation issue is about _verb versus noun_: unfortunately, "monitor", "display", and "screen" can all be both.

Author: But I hope that with display it is correct/better.

PowerToys Run enables a set of system level actions that can be executed.

> [!TIP]
> By default you can search the system commands by their localized name if your system language is supported by PowerToys application.

> > But if you prefere to search them by their english names you can enabled the non-localized search by changing the plugin setting **'Use localized system commands instead of English ones'** to **disabled** in the [plugin manager](#plugin-manager).

Reviewer 2: Prefere -> prefer

> > By default you can search the system commands by their localized name if your system language is supported by PowerToys application.
> > But if you prefere to search them by their english names you can enabled the non-localized search by changing the plugin setting **'Use localized system commands instead of English ones'** to **disabled** in the [plugin manager](#plugin-manager).

Reviewer: ```suggestion
> If the system language is supported by PowerToys, the system commands will also be localized. To use English commands, uncheck the setting **Use localized system commands instead of English ones**  in the [plugin manager](#plugin-manager).
```
\#KISS:  Keep it Short & Simple :)

Author: will be localized

if you prefer

| Command / Search result | Localized command name | Action | Note |
| :--- | :--- | :--- | :--- |

> | `Shutdown` | _Shutdown_ | Shuts down the computer | |

Author: The italic text gets translated, but not the code text between the ` signs.

> | `Shutdown` | _Shutdown_ | Shuts down the computer | |

Author: The italic text gets translated, but not the code text between the ` signs.

Reviewer: Nice try. Unfortunately, this only applis to a number of languages. Dutch is not one of them. That means I see two colums with the same text and that doesn't mean much at all. The second column is redundant.

| `Restart` | _Restart_ | Restarts the computer | |
| `Sign Out` | _Sign Out_ | Signs current user out | |
| `Lock` | _Lock_ | Locks the computer | |

> | `Sleep` | _Sleep_ | Sleeps the computer | |

Reviewer 2: Puts the computer to sleep

Author: Typo exist since ever and no one found it till now. 😅

| `Hibernate` | _Hibernate_ | Hibernates the computer | |
| `Empty Recycle Bin` | _Empty Recycle Bin_ | Empties the recycle bin | |
| `UEFI Firmware Settings` | _UEFI Firmware Settings_ | Reboot computer into UEFI Firmware Settings | Only available on systems with UEFI firmware.<br />(Requires administrative permissions.) |
| `IP address` * | _IP address_ | Shows the ip addresses from the network connections of your computer. | The search query has to start with the word `IP` or the word `address`. |

> | `MAC address` * | _MAC address_ | Shows the mac addresses from the network adapters in your computer. | The search query has to start with the word `MAC` or the word `address`. |

Author: Hopefully adding `the word` fixes translation here.

> _*) This command may take some time to provide the results._

Reviewer: ```suggestion
_\*) This command may take some time to provide the results._
```

Author: why?

Reviewer: Small thing, it works now, but idea is to prevent possible issue with formatting. (reminded me of typing a `|` inside a table cel)

Author: do we need it in the table too?

Reviewer: Nah, that's a different thing. It was just to be on the safe side (for the combination with the `_`) but it works fine either way

> [!NOTE]
> The calculator plugin respects the number format settings of your system. Please be aware of the different decimal and thousand delimiters in different locals.

> [!IMPORTANT]

> > In case your system's number format uses the comma sign as decimal delimiter you have to write a space between number and comma sign on operations with multiple parameters. Then your input has to look like this: `min( 1,2 , 3 , 5,7)` or `min( 1.2 , 3 , 5.7)`.

Reviewer: ```suggestion
> If your system's number format uses the comma (`,`) as a decimal delimiter, you have to write a space between the number(s) and comma(s) on operations with multiple parameters. The input has to look like this: `min( 1,2 , 3 , 5,7)` or `min( 1.2 , 3 , 5.7)`.
```

| Division |  a / b | |
| Modulo/Remainder | a % b | |
| Exponentiation | a ^ b | |
| Ceiling function | ceil( x.y ) |  Rounds a number up to the next larger integer. |
| Floor function | floor( x.y ) | Rounds a number down to the next smaller integer. |

> | Rounding | round( x.abcd ) | _Example: `round(8.7867)`_ |

Reviewer: Not "rounds to nearest integer" ?

Author: isn't this clear as you don't define how it should round?

| Exponential function | exp( x ) | Returns e raised to the specified power. |
| Maximum | max( x, y, z ) | |
| Minimum | min( x, y, z ) | |
| Absolute | abs( -x ) | Absolute value of a number. |
| Logarithm base 10 | log( x ) | |
| Logarithm base e | ln( x ) | |
| Square root | sqrt( x ) | |
| Power | pow( x, y ) | Calculate a number (x) raised to the power of some other number (y). |

> | Factorial | x! | |

Author: It works with and without space between number and `!`.

| Sign | sign( -x ) | A number that indicates the sign of value:<br />- `-1` if number is less than zero.<br />- `0` if number is zero.<br />- `1` if number is greater than zero. |
| Random number | rand() | Returns a fractional number between 0 and 1. |

> | Pi | pi | Returns the number of Pi.<br />_(To simply display pi, a sign is required like in this example: `+pi`.)_ |

Author: @Jay-o-Way 
I removed the `+` in the example and added an additional description.

Reviewer: Small thing: "pi" doesn't need a capital P. It's a letter, not a given name. And I think "the value of pi" sounds better, what do you think?

Author: I think it looks strang if we start with lower case letter as other start uppercase.

I think it's okay as is. I'ts a short form of `The number Pi` and Pi is the name of the number and therefore a name.

> | Pi | pi | Returns the number of Pi.<br />_(To simply display pi, a sign is required like in this example: `+pi`.)_ |

Reviewer: Small thing: "pi" doesn't need a capital P. It's a letter, not a given name. And I think "the value of pi" sounds better, what do you think?

Author: I think it looks strang if we start with lower case letter as other start uppercase.

I think it's okay as is. In this case Pi is the name of the number and therefore we can handle it as such one.

Reviewer: Or... we could make all the "actions" lowercase...

Author: let's keep as is. For me the first column is used as title and titles start uppercase.

I like to get this in now.

### Time and Date values

> The Time and Date plugin provides the current time and date or a custom one in different formats. You can enter the format or a custom time/date or both when searching.

Reviewer: This should be one plug-in, you sure both words need a Capital? The words in the [note] below should follow.

Author: my mistake. good catch. Thinking if we should write plugin names italic or bold to mark it some way? 🤔

Reviewer: On this page we have explicit headers, so therefore I would say no need.

- `( unix epoch::3/27/2022 10:30:45 AM` to convert the given time and date value into a Unix epoch timestamp.

> ### Unit conversations

Reviewer: I think it's "conversions" :P
